src/util/string.h

```diff
@@ -357,6 +357,16 @@ inline std::string to_string(T val)
 	return oss.str();
 }
 
+/// Returns A string representing the value \p val.
+template <typename T>
+inline T from_string(const std::string &str)
+{
+	std::stringstream tmp(str);
+	T t;
+	tmp >> t;
+	return t;
+}
+
```

Comment:
Okay, squashed the commit as there is only one line thats not related to the actual addition of `from_string`, there is not much sense for a separate commit here.